api/spec/requests/spree/api/taxons_controller_spec.rb

```diff
@@ -189,7 +189,7 @@ module Spree
       end
 
       it "cannot create a new taxon with invalid attributes" do
-        post spree.api_taxonomy_taxons_path(taxonomy), params: { taxon: { foo: :bar } }
+        post spree.api_taxonomy_taxons_path(taxonomy), params: { taxon: { name: '' } }
```

Comment:
I'm not sure about the reason why for this change, can you explain to me? thanks 🙏

Reply:
It was passing due the wrong check, it was actually passing because `foo` is not a permitted param, and using name it actually runs the validations